cmd/orb.go

```diff
@@ -533,6 +552,38 @@ func publishOrb(opts orbOptions) error {
 	return nil
 }
 
+func setOrbListStatus(opts orbOptions) error {
+	var err error
+
+	namespace, orb, err := references.SplitIntoOrbAndNamespace(opts.args[0])
+
+	if err != nil {
+		return err
+	}
+
+	unlist, err := strconv.ParseBool(opts.args[1])
+	if err != nil {
+		return errors.New("Specify \"true\" or \"false\" to set whether the orb should be unlisted or not")
```

Comment:
It would be useful to echo the input `opts.args[1]` back to the user in this error message, in case they have got the order or parameters incorrect. Something like:

> Expected "true" or "false", got "acme/super-orb"

Reply:
Thanks @marcomorain , I agree! I've updated the code and the validation screenshot accordingly 🙂